src/modules/StorageProposal.rb

```diff
@@ -668,7 +668,7 @@ def try_add_boot(conf, disk, force)
         pb = {}
 	pb["mount"] = Partitions.BootMount
 	pb["size"] = Partitions.ProposedBootsize
-	if disk.fetch("label","")=="gpt" && !Partitions.EfiBoot
+	if disk.fetch("label","")=="gpt" && !Partitions.EfiBoot && !is_dasd?(disk.fetch("device", ""))
```

Comment:
>  change the boot size to 1 MiB on non-EFI system on disks with GPT

Yet I see `sz = 200*1024 if sz<200*1024` below which I interpret as "make the size at least 200*1024 KiB" and that confuses me.